### PR TITLE
Enable ansible-lint

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Enable ansible-lint

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: .yamllint [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
+  braces:
+    max-spaces-inside: 1
+    level: error
+  brackets:
+    max-spaces-inside: 1
+    level: error
   document-start: disable
-
+  line-length: disable
+  truthy: disable
```
Comment (abikouo) [inline .yamllint:13]: can we remove this? it is a good ansible practice to restrict boolean values to `true` and `false`

File: changelogs/fragments/linting.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+minor_changes:
```
Comment (abikouo) [inline changelogs/fragments/linting.yml:1]: not sure if we really need a changelog as the collection has no release yet

**Review by abikouo — APPROVED** [review]

**Comment by gravesm** [comment]
It looks like we still need the zuul config for this repo. Ideally, we should get that set up before merging this.

**Comment by alinabuzachis** [comment]
> It looks like we still need the zuul config for this repo. Ideally, we should get that set up before merging this.

I opened these to set that:

https://github.com/ansible/ansible-zuul-jobs/pull/1664
https://github.com/ansible/zuul-config/pull/458
https://github.com/ansible/zuul-config/pull/459

File: .yamllint (hunk 1 above)
Comment (alinabuzachis) [reply .yamllint:13]: Allowed values should also be False and True. By default truthy allows only ['true', 'false']. At this point, not sure if we define a custom list of allowed values or keep it disabled.

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Pull request merged**